public/app/core/components/PageActionBar/PageActionBar.tsx

```diff
@@ -24,6 +39,13 @@ export default class PageActionBar extends PureComponent<Props> {
         <div className="gf-form gf-form--grow">
           <FilterInput value={searchQuery} onChange={setSearchQuery} placeholder={placeholder} />
         </div>
+        {sortPicker && (
+          <SortPicker
+            onChange={sortPicker?.onChange}
```

Comment:
Do we need the `?.` optional chaining here even if we checked if `sortPicker` is not falsy before?

Reply:
Done